portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/databasepartitioning/DatabasePartitioning.testcase

```diff
@@ -217,6 +217,27 @@ definition {
 			expectedCount = "1");
 	}
 
+	@description = "This is a use case for LPS-160064."
+	@priority = "3"
+	test CanAddAVirtualInstanceWithDBPartitioningEnabled {
+		property custom.properties = "virtual.hosts.valid.hosts=localhost,127.0.0.1,www.baker.com";
```

Comment:
Is this property necessary in this test? Looks like there is no other Database Partitioning test set up this property.

Reply:
In fact, it doesn't seem necessary